src/huggingface_hub/commands/upload.py

```diff
@@ -215,7 +216,12 @@ def _upload(self) -> str:
             if self.delete is not None and len(self.delete) > 0:
                 warnings.warn("Ignoring `--delete` since a single file is uploaded.")
 
-        if not HF_HUB_ENABLE_HF_TRANSFER:
+        if not is_xet_available():
+            logger.info(
```

Comment:
Isn't it a bit too early to add this log for all users? Given that Xet is currently enabled only for a small subset of them?

Reply:
Thanks!